org.jacoco.core.test/src/org/jacoco/core/internal/instr/NoneProbeArrayStrategyTest.java

```diff
@@ -0,0 +1,49 @@
+/*******************************************************************************
+ * Copyright (c) 2009, 2019 Mountainminds GmbH & Co. KG and Contributors
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *    Evgeny Mandrikov - initial API and implementation
+ *
+ *******************************************************************************/
+package org.jacoco.core.internal.instr;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.objectweb.asm.tree.ClassNode;
+
+public class NoneProbeArrayStrategyTest {
+
+	private NoneProbeArrayStrategy strategy;
+
+	@Before
+	public void setup() {
+		strategy = new NoneProbeArrayStrategy();
+	}
+
+	@Test
```

Comment:
Why not `@Test(expected=UnsupportedOperationException.class)`?

Reply:
@marchof for consistency with many other tests

Reply:
Hm, if we don't need additional verification we typically use annotation, e.g. `ClassInstrumenterTest`.

Reply:
changed